mergify_cli/__init__.py

```diff
@@ -701,6 +715,7 @@ def format_pull_description(message: str, depends_on: PullRequest | None) -> str
         depends_on_header = f"\n\nDepends-On: #{depends_on['number']}"
 
     message = CHANGEID_RE.sub("", message).rstrip("\n")
+    message = DEPENDS_ON_RE.sub("", message).rstrip("\n")
```

Comment:
I don't get it. It seems like we remove the Depends-On every time, no?

Reply:
We remove existing Depends-On, to replace them with the good one.
This is exactly to fix the issue you have seen these days. Like this one: https://mergify.slack.com/archives/C02R12RKPV1/p1715868421899109

Reply:
Oh, OK, I get it.

Reply:
Oh, OK, I get it.